src/components/intro/style.scss

```diff
@@ -47,7 +49,7 @@
 }
 
 .open-image-guide {
-  font: 300 11vw intro-text;
+  font: 300 11vw intro-text,helvetica,arial;
```

Comment:
FWIW, `sans-serif` has the same effect as `helvetica,arial`.

Reply:
iirc just helvetica would as well, since windows substitutes Arial anyway. good catch.